package.json

```diff
@@ -18,5 +18,8 @@
     "node-fetch": "^2.6.7",
     "trim": "^0.0.3",
     "prism": "^1.27.0"
+  },
```

Comment:
Following Monorepo approach: https://github.com/storybookjs/storybook/issues/15336#issuecomment-869472304
Other method of `@storybook/react-docgen-typescript-plugin` disabling `typescript: { reactDocgen: false },` is not ideal as this was causing issues with our React Docs because it disables Set String and converts everything to Set Objects which will cause the docs to fail in certain instances